src/views/CardStack.js

```diff
@@ -207,11 +224,16 @@ class CardStack extends React.Component<DefaultProps, Props, void> {
           if (header && header.left) {
             return header.left;
           }
-          const { renderLeftComponent } = this.props.headerComponent.defaultProps || {};
-          if (typeof renderLeftComponent === 'function') {
-            return renderLeftComponent(props);
+          if (props.scene.index === 0 || !props.onNavigateBack) {
+            return null;
           }
-          return null;
+          const prevSceneTitle = this._getHeaderTitleForScene(props.scenes[props.index - 1]);
+          return (
+            <HeaderBackButton
```

Comment:
> but shouldn't any custom header component be able to render the back button with title?

If you define a custom header, the `this.props.renderLeftComponent(props)` will return you a standard back button anyway as long as your route doesn't override it. 

> if I want to use custom header button I can't show the title

If you want to show a custom header button (`left`), it's already up to you to access the title yourself I believe.

> if I want to show the title I can't use a custom back button

Not sure I understand? The default back button with a title will not show only if either of these is true:
- you have a custom `left` component
- you have a non-string `title` 

> How about a prop to header called getBackTitleForScene

How that improves over the existing interface? All you are saying is the ability to still have an access for a backButtonTitle when `header.left` is specified?